views/default/object/elements/summary/metadata.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Outputs object metadata
+ * @uses $vars['metadata'] Metadata/menu
+ */
+
+$metadata = elgg_extract('metadata', $vars);
+if (!$metadata) {
+	return;
+}
+echo $metadata;
```

Comment:
Ideally we should wrap this as `<div class="elgg-list-summary-metadata">` and float it instead of floating the entity menu (makes it hard to display an entity menu elsewhere on the page without having to restyle it).